Update setting_primary_tags_to_scope.md (#28881)
DataDog/documentation: content/en/tracing/guide/setting_primary_tags_to_scope.md

## Add additional primary tags in Datadog

> If you need to aggregate your trace metrics across additional dimensions, Datadog recommends setting up additional primary tags in addition to the mandatory primary tag `env:<ENVIRONMENT>`. Once configured, primary tags will become available on service pages and on the **Software Catalog Performance** tab as a filter

Reviewer: ```suggestion
If you need to aggregate your trace metrics across additional dimensions, Datadog recommends setting up additional primary tags in addition to the mandatory primary tag `env:<ENVIRONMENT>`. After those are configured, primary tags are available on service pages and on the **Software Catalog Performance** tab as a filter
```
